### ISSUE TITLE
Renode tests of CMSIS-NN kernels tests are not working for all

### ISSUE THREAD

**Issue by mansnils** [tensorflow/tensorflow#44455]
@tensorflow/micro

**System information**
- Host OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):
- Tensorflow version (commit SHA if source): 5b5960f4fda6a6ae9cbf5233873b9ea6910b3e4e
- Target platform (e.g. Arm Mbed OS, Arduino Nano 33 etc.):

**Describe the problem**
CMSIS-NN kernel tests conv and softmax are not working.

**Please provide the exact sequence of commands/steps when you ran into the problem**

make -j4 -f tensorflow/lite/micro/tools/make/Makefile TAGS=cmsis-nn TARGET=stm32f4  test_kernel_conv_test
make -j4 -f tensorflow/lite/micro/tools/make/Makefile TAGS=cmsis-nn TARGET=stm32f4  test_kernel_softmax_test

**Comment by freddan80** [comment]
Can be closed now
